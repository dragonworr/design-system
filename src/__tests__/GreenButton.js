import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { GreenButton, theme } from '..'

describe('GreenButton', () => {
  test('renders', () => {
    const json = renderer.create(<GreenButton theme={theme} />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
