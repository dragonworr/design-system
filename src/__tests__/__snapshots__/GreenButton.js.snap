// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreenButton renders 1`] = `
.c1 {
  display: inline-block;
  vertical-align: middle;
  text-align: center;
  outline: none;
  text-decoration: none;
  font-weight: 600;
  text-rendering: optimizeLegibility, -webkit-font-smoothing:antialiased cursor:pointer;
  border-radius: 2px;
  background-color: #007aff;
  color: #fff;
  border-width: 0;
  border-style: solid;
  height: 40px;
  font-size: 14px;
  padding: 0 18px;
}

.c1:disabled {
  opacity: 0.25;
}

.c0 {
  background-color: #0a0;
}

<button
  className="c0 c1"
/>
`;
